torch/_dynamo/trace_rules.py

```diff
@@ -0,0 +1,48 @@
+import enum
+import functools
+import importlib
+
+
+class TraceRule(enum.Enum):
+    IN_GRAPH_FUNCTION = 0
+    CONST_FOLD_FUNCTION = 1
```

Comment:
Are IN_GRAPH and CONSTANT_FOLD mutually exclusive?

Reply:
I don't think so, probably we should have only one and map it to ```TorchVariable```. WDYT?